content/docs/command-reference/commit.md

```diff
@@ -21,9 +20,9 @@ positional arguments:
 The `dvc commit` command is useful for several scenarios, when data already
 tracked by DVC changes: when a [stage](/doc/command-reference/run) or
 [pipeline](/doc/command-reference/dag) is in development/experimentation; when
-manually editing or generating DVC <abbr>outputs</abbr>; or to force DVC-file
-updates without reproducing stages or pipelines. These scenarios are further
-detailed below.
+manually editing or generating DVC <abbr>outputs</abbr>; or to force the
+`dvc.lock` or the `.dvc` updates without reproducing stages or pipelines. These
```

Comment:
Still weird grammar 😢

Reply:
Adding "the" everywhere is not a solution for grammar issues 😄 need to read it and see if there are missing words, etc... 🙏